kubernetes/hypertrace.sh

```diff
@@ -76,22 +112,90 @@ if [[ "$HT_ENABLE_DEBUG" == "true" ]]; then
   KUBE_FLAGS="$KUBE_FLAGS --v=4"
 fi
 
+servicename=""
+
 if [ "$#" -gt 1 ]; then
   if [[ $2 == "--clean" ]]; then
       clean
-    else
-      usage
-  fi
+  else
+      servicename=$2
+    fi
 elif [ "$#" -ne 1 ]; then
     echo "[ERROR] Illegal number of parameters"
     echo "-------------------------------------"
     usage
 fi
 
-
 subcommand=$1; shift
 
 case "$subcommand" in
+  generate-manifests)
+    EXIT_CODE=0;
+    cleanup
+    cleanup_manifests
+    create_manifest_directories
+    if [[ "$servicename" != "" && $( echo "${servicename}" |  egrep -c "^(kafka|zookeeper|pinot|schema-registry|mongo|postgres)$" || :;) -ne 0 ]]; then
+      mkdir ${HYPERTRACE_HOME}/data-services/helm-deployment-templates/service-manifests/
+      echo "[INFO] creating helm deployment template for" $servicename
+      helm dependency update ${HYPERTRACE_HOME}/data-services ${HELM_FLAGS}
+      tar -xvf ${HYPERTRACE_HOME}/data-services/charts/"$servicename"* -C ${HYPERTRACE_HOME}/data-services/charts/
+      if [ $HT_DATA_STORE == "postgres" ]; then
+        helm template ${HYPERTRACE_HOME}/data-services/charts/$servicename -f ${HYPERTRACE_HOME}/data-services/values.yaml -f ${HYPERTRACE_HOME}/data-services/postgres/values.yaml -f ${HYPERTRACE_HOME}/clusters/$HT_PROFILE/values.yaml --set htEnv=${HT_ENV} > ${HYPERTRACE_HOME}/data-services/helm-deployment-templates/service-manifests/$servicename-manifests.yaml
+      else
+        helm template hypertrace-data-services ${HYPERTRACE_HOME}/data-services/charts/$servicename -f ${HYPERTRACE_HOME}/data-services/values.yaml -f ${HYPERTRACE_HOME}/clusters/$HT_PROFILE/values.yaml --set htEnv=${HT_ENV} > ${HYPERTRACE_HOME}/data-services/helm-deployment-templates/service-manifests/$servicename-manifests.yaml
+      fi
+      echo "[INFO]" $servicename " manifests are generated at: " ${HYPERTRACE_HOME}"/data-services/helm-deployment-templates/service-manifests/"$servicename"-manifests.yaml"
+      cleanup
+
+    elif [[ "$servicename" != "" ]]; then
+      helm dependency update ${HYPERTRACE_HOME}/platform-services ${HELM_FLAGS}
+        if [[ "$servicename" == "pre-install-tasks" ]]; then
```

Comment:
Here, it seems `servicename` is overloaded with pre/post install tasks.
Shall we do as below?
```
./hypertrace.sh generate-manifests --service service-name
./hypertrace.sh generate-manifests --deps pre-install-tasks
```